digdag-standards/build.gradle

```diff
@@ -15,7 +15,7 @@ dependencies {
     compile 'com.sun.mail:javax.mail:1.5.6'   // 'com.sun.mail:smtp' doesn't work because enabling mail.debug property throws java.lang.NoClassDefFoundError: com/sun/mail/util/MailLogger
 
     // td
-    compile 'com.treasuredata.client:td-client:0.8.5'
+    compile 'com.treasuredata.client:td-client:0.8.6'
```

Comment:
This update is needed to access the attribute `TDJob.numRecords`

Reply:
👍